suite-common/wallet-types/src/account.ts

```diff
@@ -42,6 +42,14 @@ type AccountNetworkSpecific =
                   isActive: boolean;
                   rewards: string;
                   poolId: string | null;
+                  drep: {
+                      drep_id: string;
+                      hex: string;
+                      amount: string;
+                      active: boolean;
+                      active_epoch: number | null;
+                      has_script: boolean;
+                  } | null;
```

Comment:
🤔 this field is not optional. doesn't it require db migration?

Reply:
I might not be seeing the full picture so I'd like to ask what the `staking` object was added to this type here https://github.com/trezor/trezor-suite/pull/16003/files#diff-590423282c67d295c7292f90e100f2863ea22b056e49ae02f135a5840475fb9bR172 (actually full `misc` object). was it a bug? how come anybody did not miss it there before?

Reply:
Yes it was missing.